lib/improver/ensemble_calibration/ensemble_calibration.py

```diff
@@ -75,7 +75,7 @@ class ContinuousRankedProbabilityScoreMinimisers(object):
     # as part of the minimisation.
     BAD_VALUE = np.float64(999999)
 
-    def __init__(self, max_iterations=1000):
+    def __init__(self, max_iterations=1000, decimals=5):
```

Comment:
~10 lines above: how did you reach a 5% tolerance for the final iteration?  This is extremely high.

Reply:
Something for future plans but not this PR.